chanacceptor/rpcacceptor.go

```diff
@@ -331,6 +331,11 @@ func (r *RPCAcceptor) sendAcceptRequests(errChan chan error,
 				):
 					commitmentType = lnrpc.CommitmentType_ANCHORS
 
+				case channelFeatures.OnlyContains(
+					lnwire.SimpleTaprootChannelsRequiredStaging,
```

Comment:
I'm not 100% sure about this as I'm not certain what flags would be set for a taproot chan? Do we support zero conf and SCID alias features as well?
Also, do taproot channels support thaw heights?

Reply:
> Also, do taproot channels support thaw heights?

Currently, no.

Reply:
We'll want to copy the features as exists here: https://github.com/lightningnetwork/lnd/blob/a1b0a7eb027ce84a05360475164e95daf201796d/funding/commitment_type_negotiation.go#L243-L308 

IMO we should consolidate these sections as well.

Reply:
Okay, updated with the zero conf and scid features. Hope this looks good.